src/App.tsx

```diff
@@ -5,15 +5,20 @@ import routes from './router'
 
 import { CssBaseline } from '@mui/material'
 import ThemeProviderWrapper from './theme/ThemeProvider'
+import React, {useState} from 'react'
+import {UserContext} from './content/Auth/UserContext'
```

Comment:
It seems the file wasn't formatted correctly as imports usually have a space around the braces.

Reply:
Fixed.